### PR TITLE
feat: 4326 scale sequencer http server

### PR DESCRIPTION
**Pull request by preethamr**
## Description
4326 scale sequencer http server

highlights:
- http server is a separate entity, just validate data and queue, nothing else
- a new queue `http` is introduced, as task queue for the "http subscriber"
- the current workload of `publisher` is split between `server` and `publisher`, with this queue in between
- tests
- terraform
- docker
- docker compose
- setup/integrations scripts
- seq MQ config changes, increase max limit to 100K
- fix bugs from changing client lib from `foofoomq` to `amqplib` (possible there are other regression bugs)

## Type of change

- [ ] Docs change / dependency upgrade
- [x] Configuration / tooling changes
- [x] Refactoring
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires changes in customer code

## Related Issue(s)

Fixes

### CODE REVIEW

**Review by rhlsthrm — DISMISSED** [review]

**Review by sanchaymittal — APPROVED** [review]
reviewed on high level, lg!

File: packages/agents/sequencer/src/bindings/server/index.ts [hunk 1]
```diff
@@ -111,14 +112,29 @@ export const bindServer = async (): Promise<FastifyInstance> => {
       try {
         const bid = request.body;
         requestContext.transferId = bid.transferId;
-        await storeFastPathData(bid, requestContext);
+        // Validate Input schema
+        const validateInput = ajv.compile(BidSchema);
```
Comment (rhlsthrm) [inline packages/agents/sequencer/src/bindings/server/index.ts:116]: There's actually an easier way to do this schema validation directly through Fastify: https://www.fastify.io/docs/latest/Reference/Validation-and-Serialization/

We are doing this in other HTTP handlers you can look at.

File: packages/agents/sequencer/src/bindings/server/index.ts [hunk 2]
```diff
@@ -111,14 +112,29 @@ export const bindServer = async (): Promise<FastifyInstance> => {
...
+        const validInput = validateInput(bid);
+        if (!validInput) {
+          const msg = validateInput.errors?.map((err: any) => `${err.instancePath} - ${err.message}`).join(",");
+          throw new DataInvalid({
```
Comment (rhlsthrm) [inline packages/agents/sequencer/src/bindings/server/index.ts:120]: Schema validation errors should not throw 500, I think using the validation strategy above will take care of that.

**Review by rhlsthrm — CHANGES_REQUESTED** [review]
Looks great! Minor change requested.

File: packages/agents/sequencer/src/bindings/server/index.ts (hunk 1 above)
Comment (preethamr) [reply packages/agents/sequencer/src/bindings/server/index.ts]: Makes sense. Done

**Pull request merged**